AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -601,25 +601,20 @@ - (void)__setNeedsLayout
     // This is the root node. Trigger a full measurement pass on *current* thread. Old constrained size is re-used.
     [self measureWithSizeRange:oldConstrainedSize];
 
-    CGSize oldSize = self.bounds.size;
+    CGRect oldBounds = self.bounds;
+    CGSize oldSize = oldBounds.size;
     CGSize newSize = _layout.size;
     
     if (! CGSizeEqualToSize(oldSize, newSize)) {
```

Comment:
I forgot about the logic duplication in this file.  Because this code path will be called dramatically less frequently than setFrame:, I agree the cleanup is a good idea.

Just revert the other file.